dynamax/hmm/models/bernoulli_hmm.py

```diff
@@ -29,6 +29,10 @@ def __init__(self,
         self.emission_prior_concentration0 = emission_prior_concentration0
         self.emission_prior_concentration1 = emission_prior_concentration1
 
+    @property
+    def emission_shape(self):
+        return (self.emission_dim,)
+
     def emission_distribution(self, params, state):
         return tfd.Independent(tfd.Bernoulli(probs=params['emissions']['probs'][state]),
                                reinterpreted_batch_ndims=1)
```

Comment:
Can you add a comment explaining the `reinterpreted_batch_ndims=1` arg?

Reply:
Done.